compile.sh

```diff
@@ -627,11 +628,29 @@ if [ "$COMPILE_GD" == "yes" ]; then
 	make install >> "$DIR/install.log" 2>&1
 	cd ..
 	echo " done!"
+	#lipjpeg
+	echo -n "[lipjpeg] downloading $LIBJPEG_VERSION..."
```

Comment:
typo, I think you mean `libjpeg`

Reply:
oh, I missed that one
(I also had a habit of writing LIBJEPG and other variations...)